src/main/scala/io/snyk/plugin/depsource/DepTreeProvider.scala

```diff
@@ -53,7 +67,7 @@ private class ProjectDepTreeProvider(project: Project) extends DepTreeProvider {
     } else if (isGradleProject) {
       getGradleBuildToolProjects
     } else {
-      NotificationService.showWarning(project, "Project type is not supported")
+      logger.info("Project type is not supported.")
```

Comment:
@aldanchenko, this will be shown in `Event Log` tab, correct?

Reply:
@pavel-snyk no, it will print them to IntelliJ log file.